src/views/credits/people.json

```diff
@@ -1,242 +1,242 @@
 [
     {
-        "userName": "labdalla",
-        "userId": 35687410,
-        "name": "Lena"
+        "userName": "",
+        "userId": "",
+        "name": "Abisola"
     },
     {
-        "userName": "shaanmasala",
-        "userId": 29995562,
-        "name": "Yusuf"
+        "userName": "amylaser",
+        "userId": 17462181,
+        "name": "Amy"
     },
     {
-        "userName": "ceebee",
-        "userId": 2755634,
-        "name": "Christan"
+        "userName": "thisandagain",
```

Comment:
Should his name be here? He left the Scratch Team.

Reply:
Thanks for asking -- checking, will get back to you